mirakuru/__init__.py

```diff
@@ -20,7 +20,7 @@
 
 import logging
 
-from mirakuru.base import Executor
+from mirakuru.base import Executor, StartCheckExecutor
```

Comment:
Actually... I'd rename Executor into SimpleExecutor and StartCheck into Executor itself.

I think it's main feature of mirakuru, that the executor doesn't declare something as running, just because it's called command. Having it as StarCheckExecutor makes it look like a secondary feature.